radio/src/targets/horus/hal.h

```diff
@@ -282,10 +282,17 @@
   #define ADC_RCC_AHB1Periph            (RCC_AHB1Periph_GPIOA | RCC_AHB1Periph_GPIOC | RCC_AHB1Periph_GPIOF | RCC_AHB1Periph_DMA2)
   #define ADC_RCC_APB1Periph            (RCC_APB1Periph_TIM5)
   #define ADC_RCC_APB2Periph            (RCC_APB2Periph_ADC1 | RCC_APB2Periph_ADC3)
-  #define ADC_GPIO_PIN_STICK_LH         GPIO_Pin_0      // PA.00
-  #define ADC_GPIO_PIN_STICK_LV         GPIO_Pin_1      // PA.01
-  #define ADC_GPIO_PIN_STICK_RH         GPIO_Pin_2      // PA.02
-  #define ADC_GPIO_PIN_STICK_RV         GPIO_Pin_3      // PA.03
+  #if !defined(FLYSKY_HALL_STICKS)
+    #define ADC_GPIO_PIN_STICK_LH         GPIO_Pin_0      // PA.00
+    #define ADC_GPIO_PIN_STICK_LV         GPIO_Pin_1      // PA.01
+    #define ADC_GPIO_PIN_STICK_RH         GPIO_Pin_2      // PA.02
+    #define ADC_GPIO_PIN_STICK_RV         GPIO_Pin_3      // PA.03
+  #else
+    #define ADC_GPIO_PIN_STICK_LH           0
```

Comment:
If possible, please try to completely undef these.

Reply:
I will see if I can work out something more clean.

Reply:
Just to say, many places rely on patterns like `#if defined(SOME_PIN_PORT)`, which would be defeated by such a definition set to `0` (still defined).

Reply:
I have changed the way to disable ADC, please check if it is ok now.

Reply:
Thx! we're good to go!